src/js/apps/globals/nav_app.js

```diff
@@ -132,8 +141,8 @@ export default App.extend({
     const currentUser = Radio.request('bootstrap', 'currentUser');
     if (!currentUser.can('admin')) appNavMenu.remove('AdminApp');
 
-    // If there's only 2 options, only sign out
-    if (appNavMenu.length === 2) appNavMenu.shift();
+    // If there's only 3 options, only sign out
+    if (appNavMenu.length === 3) appNavMenu.shift();
```

Comment:
what is this change?

Reply:
Yeah, I thought this seemed a little fragile. This is removing the "Workspace" menu item if the user isn't an admin. Although, it could be cleaned up by adding `appNavMenu.remove('PatientsApp')` in the `currentUser.can('admin')` check above.